src/fidesops/schemas/saas/saas_config.py

```diff
@@ -45,8 +52,8 @@ class SaaSRequest(BaseModel):
     path: str
     request_params: Optional[List[RequestParam]]
     data_path: Optional[str]  # defaults to collection name if not specified
-    preprocessor: Optional[Strategy]
-    postprocessor: Optional[List[Strategy]]
+    preprocessors: Optional[Strategy]
```

Comment:
I know we aren't using this yet. Should this be a list also?

Reply:
yup, probably should!